build-system/compile/compile-wrappers.js

```diff
@@ -58,19 +87,69 @@ exports.extension = function (name, version, latest, isModule, loadPriority) {
   // Use a numeric value instead of boolean. "m" stands for "module"
   const m = isModule ? 1 : 0;
   return (
-    `(self.AMP=self.AMP||[]).push({n:"${name}",ev:"${version}",l:${latest},` +
-    `${priority}` +
-    // The `function` is wrapped in `()` to avoid lazy parsing it, since it will
-    // be immediately executed anyway.
-    // See https://github.com/ampproject/amphtml/issues/3977
-    // TODO(wg-performance): At some point, the build pipeline began stripping
-    // out these parentheses. Is this optimization still relevant?
-    `v:"${VERSION}",m:${m},f:(function(AMP,_){\n` +
-    '<%= contents %>\n})});'
+    '{' +
+    `m:${m},` +
+    `v:"${VERSION}",` +
+    `n:"${name}",` +
+    `ev:"${version}",` +
+    `l:${latest},` +
+    priority +
+    `f:(function(AMP,_){<%= contents %>})` +
+    '}'
+  );
+}
+
+/**
+ * Anonymous function to load a Bento extension's payload (p).
+ * @see {@link bento}
+ */
+const bentoLoaderFn = removeWhitespace(`
+function (p) {
```

Comment:
nit: Can you convert this to if/else instead of a double ternary?

Reply:
Using `if/else`, we get slightly larger output. It looks like we run a minifier step on the wrapped binary, but it seems to work less aggressively than for the rest of the content. 

```diff
- function(p){self.AMP?self.AMP.push(p):document.head.querySelector('script[src$="v0.js"],script[src$="v0.mjs"]')?self.AMP=[p]: (...)
+ function(p){if(self.AMP)self.AMP.push(p);else if(document.head.querySelector('script[src$="v0.js"],script[src$="v0.mjs"]'))self.AMP=[p];else (...)
```